editor/assets/stylesheets/_z-index.scss

```diff
@@ -13,8 +13,8 @@ $z-layers: (
 	'.editor-post-visibility__dialog': 30,
 	'.editor-post-schedule__dialog': 30,
 	'.editor-block-mover': 30,
-	'.components-notice-list': 100000,
-	'.components-popover': 100000,
```

Comment:
I don't think so - the notice is part of the Gutenberg UI, not something that should block you from using other wp-admin functions.

The design of traditional WP notices is very different (inline with the page, rather than popovers) but they don't suffer from this issue.